### ISSUE TITLE
Problem with setting up "Caching Docker Image on an EBS Volume"

### ISSUE THREAD

**Issue by qraleq** [spotty-cloud/spotty#78]
Hi, I set up everything according to the instructions related to "Caching Docker Image on an EBS Volume", and everything worked perfectly in the previous version of Spotty, but now in 1.3.0 I have problems starting an instance using the same settings.

Spotty throws the following error:
`Validation error: The "mountDir" of one of the volumes must be a prefix for the "dockerDataRoot" path.`

Can you please help me resolve this?

**Comment by apls777** [comment]
Thank you, Ivan. It's a bug. A quick workaround would be to explicitly specify mount directories for the rest of the instance volumes. For example:
```
...
dockerDataRoot: /docker
volumes:
  - name: project
    parameters:
      size: 20
      mountDir: /mnt/project
  - name: data
    parameters:
      size: 500
      mountDir: /mnt/data
  - name: docker
    parameters:
      size: 10
      mountDir: /docker
```

**Comment by qraleq** [comment]
Thanks, Oleg!

**Comment by apls777** [comment]
Fixed in v1.3.1
